core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -117,6 +119,13 @@ statement
     | CALL qualifiedName '(' (callArgument (',' callArgument)*)? ')'   #call
     | CREATE (OR REPLACE)? functionSpecification                       #createFunction
     | DROP FUNCTION (IF EXISTS)? functionDeclaration                   #dropFunction
+    | CREATE (OR REPLACE)? BRANCH (IF NOT EXISTS)? identifier
+        (WITH properties)? IN TABLE qualifiedName                      #createBranch
+    | DROP BRANCH (IF EXISTS)? identifier
+        IN TABLE qualifiedName                                         #dropBranch
+    | ALTER BRANCH source=identifier IN TABLE qualifiedName
```

Comment:
```
ALTER BRANCH source=identifier IN TABLE qualifiedName
        FAST FORWARD TO target=identifier                              #fastForwardBranch
```

When we want the changes from the branch `experiment` in `main` does this correspond to

```
ALTER BRANCH main IN TABLE test-table
        FAST FORWARD TO experiment
```

?

Reply:
Yes, the query will call `ManageSnapshots#fastForwardBranch("main", "experiment")` in Iceberg connector.